bluetooth-CC2650-demo/README.md

```diff
@@ -0,0 +1,115 @@
+# Bluetooth With KubeEdge Using CC2650
+
+
+Users can make use of KubeEdge platform to connect and control their bluetooth devices, provided, the user is aware of the of the data sheet information for their device.
+Kubernetes Custom Resource Definition (CRD) and KubeEdge bluetooth mapper is being used to support this feature, using which users can control their device from the cloud. Texas Instruments CC2650 SensorTag device is being shown here as an example.
```

Comment:
It's good if we can give a link to the CC2650 device product page or datasheet for users to check out.

Reply:
Done, thanks